src/client.js

```diff
@@ -1137,6 +1137,33 @@ function SugarApi(args) {
             return this.call(method, url, data.related, callbacks, options);
         },
 
+        /**
+         * Fetch the lean count of related records.
+         *
+         * The data paramerer represents relationship information:
+         * <pre>
+         * {
+         *    id: record ID
+         *    link: relationship link name
+         *    relatedId: ID of the related record
+         *    related: object that contains request payload (related record or relationship fields)
+         * }
+         * </pre>
+         *
+         * @param {string} module module name.
+         * @param {Object} data object with relationship information.
+         * @param {Object} [params] URL parameters.
+         * @param {Object} [callbacks] callback object.
+         * @param {Object} [options] request options.
+         * @return {HttpRequest} The AJAX request.
+         * @memberOf Api
+         * @instance
+         */
+        relatedLeanCount: function(module, data, params, callbacks, options) {
+            var url = this.buildURL(module, 'leancount', data, params);
+            return this.call('read', url, data.related, callbacks, options);
```

Comment:
It's using read because that's the only method we support for this end point. But I understand the concern about flexibility of ventana. Let me think again to see if makes sense to support others.

Reply:
I think I'd like to keep it this way because:
1). This is to get count. I can't think of any use case that you would want to create count, update count or delete count. That doesn't make sense to me.
2). On the server side, the end point doesn't support other methods at least for now. So if we allow user to call this with other method, it will result in failure.

Reply:
Ok. Makes sense. Thanks for the explanation.